tests/Compat/FormDecorator/LabelDecoratorTest.php

```diff
@@ -23,4 +27,52 @@ public function testWithoutLabelAttribute(): void
 
         $this->assertSame('&nbsp;', $results->assemble()->render());
     }
+
+    public function testWithRequiredField(): void
```

Comment:
You should include an additional case:

- decorate a required elemement
- decorate a non-required element
- decorate the form
- assert that the form-info is rendered

We're dealing with internal state after all, so let's make sure it works as designed :)